README.md

```diff
@@ -27,7 +27,9 @@ Persistent storage is configured using the configmap `cluster-monitoring-config`
 
 ## SRE Authorization
 
-Instead of SRE having `cluster-admin` a new ClusterRole is created with some permissions removed.  The ClusterRole can be regenerated in the `generate/sre-authorization` directory.
+Instead of SRE having `cluster-admin` a new ClusterRole `osd-sre-admin` is created with some permissions removed.  The ClusterRole can be regenerated in the `generate/sre-authorization` directory.  The role is granted to SRE via `osd-sre-admins` group.
```

Comment:
```suggestion
Instead of SRE having the `cluster-admin` role, a new ClusterRole, `osd-sre-admin`, is created with some permissions removed.  The ClusterRole can be regenerated in the `generate/sre-authorization` directory.  The role is granted to SRE via `osd-sre-admins` group.
```

Editing